doc/source/ray-core/ray-dashboard.rst

```diff
@@ -322,6 +545,11 @@ Actors
 
 Logs
```

Comment:
"Logging structure"?
Maybe we should combine it with the other sections ("the guide for viewing application logs" and the "logs view").

Reply:
it's the reference page. I think I will just remove all the reference page later (we should just comment it within the UI)

Reply:
For now, I will just delete it as it is not that useful